cmd/node/factory/structs.go

```diff
@@ -147,7 +147,7 @@ type processComponentsFactoryArgs struct {
 	version                   string
 	importStartHandler        update.ImportStartHandler
 	workingDir                string
-	historyProc               fullHistory.HistoryHandler
+	historyRep                fullHistory.HistoryRepository
```

Comment:
If using the abbreviation, then `repo` instead of `rep`.

Reply:
done.